pages/phases/certificate.md

```diff
@@ -20,16 +20,27 @@ roles: [hr, hiring-mgr]
   {% include toolkit/certificate.html %}
 </div>
 
-## HR Preference Adjudication
+<div class="usa-alert usa-alert--info" >
+  <div class="usa-alert__body">
+    <h3 class="usa-alert__heading">Do not conduct a separate minimum qualification review</h3>
+    <p class="usa-alert__text">
+      After SMEs have completed interviews and identified the applicants who are found to meet qualifications, do not conduct a separate minimum qualification review. For more information, see "<a href="{{ site.baseurl }}/about/differences/#sme-assessment">How our process is different</a>."
+    </p>
+  </div>
+</div>
+
+## Adjudicating Preference
+
+Apply Veterans' preference only to applicants who achieve an overall passing score. Up until that point, any applicant can be removed from consideration if they are determined not to qualify based on the required competencies and proficiencies.
 
-Veterans' Preference will be applied only to those who meet the overall passing score. Up until that point, any applicant can be removed from consideration if they are determined not to meet the required qualifications (competencies/proficiencies).
+OPM's MSAC ACE will review the final category assignments to ensure they're lawful. HR will then issue the certificate of eligibles of the "Best Qualified" category to the HM.
```

Comment:
actually MSAC won't do this - remove- that was just for the pilot

Reply:
why are we removing the red part?

Reply:
> why are we removing the red part?

We're not, it just moved to another location. You reviewed it above.